Jenkinsfile

```diff
@@ -49,7 +49,7 @@ pipeline {
                 env.BRANCH_NAME = ""
             }
             env.GIT_COMMIT = bat(returnStdout: true, script: '@git rev-parse HEAD').trim()
-            env.GIT_BRANCH = bat(returnStdout: true, script: '@git rev-parse --abbrev-ref HEAD').trim()
+            env.GIT_BRANCH = "origin/" + scm.branches[0].name
             echo "git commit: ${env.GIT_COMMIT}"
```

Comment:
Does the existing check for env.BRANCH_NAME need removing, or should this code go into the if clause? Has it been checked on multibranch pipelines?